src/modules/ROOT/pages/changelog/202303/01.json

```diff
@@ -26,9 +23,29 @@
         "to give users the ability to open records in a modal without leaving the page that they are on."
       ],
       "refs": [
-        "https://github.com/cortezaproject/corteza/pull/1177/commits/ae2e908f9307baa329b43681d43af5492dc198ee",
-        "https://github.com/cortezaproject/corteza/pull/1177/commits/8508eb37d1f60da14955973294147897ee7fefb9",
-        "https://github.com/cortezaproject/corteza/pull/1177/commits/a62f60fd3bc6b55f5896e95626080438737f920a"
+        "https://github.com/cortezaproject/corteza/pull/1177"
+      ]
+    },
+    {
+      "what": [
+        "support for error handling message inside workflows by letting configurators access the context of the error message."
+      ],
+      "why": [
+        "to give user more freedom in defining error handling and to know where and on which step the error occurred."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/1088"
+      ]
+    },
+    {
+      "what": [
+        "http prefixes to all URL input values across {PRODUCT_NAME} if it is not provided by the user."
```

Comment:
i'm quite sure it's https not http; fact-check with the author

Reply:
Firstly I wrote it https but after checking with the author he corrected me that it needs to be http. So all good.